lib/core/services/auth-guard-base.ts

```diff
@@ -91,4 +91,14 @@ export abstract class AuthGuardBase implements CanActivate, CanActivateChild {
             this.authenticationService.isOauth() && oauth.silentLogin === false
         );
     }
+
+    protected isOAuthWithSilentLogin() {
+        const oauth = this.appConfigService.get<OauthConfigModel>(
+            AppConfigValues.OAUTHCONFIG,
+            null
+        );
+        return (
```

Comment:
you get `AppConfigValues.OAUTHCONFIG` or `null`. missing the null check in the code below, may crash at runtime